[2025-03-17T20:40:34.121+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-17T20:40:36.840+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dag.load_tables.pages_project_b_db scheduled__2025-03-17T20:30:00+00:00 [queued]>
[2025-03-17T20:40:39.151+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dag.load_tables.pages_project_b_db scheduled__2025-03-17T20:30:00+00:00 [queued]>
[2025-03-17T20:40:39.161+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-03-17T20:40:40.671+0000] {taskinstance.py:2889} INFO - Executing <Task(PostgresOperator): load_tables.pages_project_b_db> on 2025-03-17 20:30:00+00:00
[2025-03-17T20:40:41.411+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=3811) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-17T20:40:41.520+0000] {standard_task_runner.py:72} INFO - Started process 3840 to run task
[2025-03-17T20:40:41.471+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_dag', 'load_tables.pages_project_b_db', 'scheduled__2025-03-17T20:30:00+00:00', '--job-id', '275', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmpufectkf9']
[2025-03-17T20:40:41.590+0000] {standard_task_runner.py:105} INFO - Job 275: Subtask load_tables.pages_project_b_db
[2025-03-17T20:40:44.671+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_dag.load_tables.pages_project_b_db scheduled__2025-03-17T20:30:00+00:00 [running]> on host c032966e105e
[2025-03-17T20:40:48.319+0000] {job.py:229} INFO - Heartbeat recovered after 19.97 seconds
